Staging_Dev/Zone_16_invalid_entities__2020_09_16.sql

```diff
@@ -0,0 +1,11 @@
+USE [perpetuumsa]
+GO
+
+-----------------------------------------------
+-- Some zone entities left over in Zone 16
+-- throwing exceptions during server startup
+--
+-- Date: 2020/09/16
+-----------------------------------------------
+
+DELETE FROM zoneentities WHERE id IN (1528, 1529, 1539)
```

Comment:
I know its a bit much, but can you look these up by name (or some unique varchar col)?
It is a baseline assumption, even if these records are from 'vanilla', that table id's are not to be relied on for stable migrations.

Reply:
Also a convention to add `GO` to the end of any singular patch file as it closes scope.  This assists with avoiding conflicts, namespace issues, etc. when rollups are done.